BUILD.linux.md

```diff
@@ -93,6 +93,12 @@ Optionally, enable auto-complete in Bash:
 source <multipass>/completions/bash/multipass
 ```
 
+To be able to use the binaries without specifying their path:
+
+```
+export PATH=<build-dir>/bin
```

Comment:
The instructions create an in-source build and so use `<multipass>/build` for the build directory in other places. I am thinking this should follow that tune here too. WDYT?